apps/site/assets/css/_utilities.scss

```diff
@@ -225,6 +225,10 @@
   color: $brand-primary-light;
 }
 
+.u-color-brand-primary {
```

Comment:
I'm not opposed to keeping this as it's more legible IMO, but just wanted to share that `.text-primary` does the same thing (thanks Bootstrap 4!)

Reply:
That explains why I couldn't find a utility class.  I always forget what bootstrap classes we have access to >.<